Doc/configuration.md

```diff
@@ -249,12 +273,55 @@ See [6. Configure the tags](#6-configure-the-tags)
  
 ## 6. Configure the tags
 
+### :warning: Modifying configuration requires statsd_exporter reboot
+
+Once you've sent a metric, if you change its configuration, changes will be ignored. 
+
+You will need to reboot the statsd_exporter server in order to take into account the new changes.
+
+
+### :oncoming_police_car::oncoming_police_car: Tag naming rules and default configuration 
+
+In order to have common metric names amongst all of our applications, we have defined some common tags names.
+If you need to get one of this data, please use the following name for your tag.
+
+#### Global configuration
+ 
+* `project`: __\[required\]__ 
+Your project name. *(service_6play_users_cloud, service_6play_middleware, ...)*
+
+#### Group configuration
+
+None yet.
+
+#### Metric configuration
+
+* `customer`: default value : *default*. Can be *all* for non-related customer applications.
+* `client`:  (UserBundle) 
+* `platform`: default value : *all* 
+* :warning: __Waiting for validation__ `release`: Value matching with the header X-Client-Release: 
+This will create TOO MUCH metrics 
+* `service`: (???)
+* HTTP Request
+   * `route`: Symfony route name *(status, get_subscriptions, etc.)*
+   * `status`: HTTP response code
+* HTTP Request to an external service
+   * `route`: Symfony route name *(status, get_subscriptions, etc.)*
+   * `status`: HTTP response code
+   * `project_to`: external service called
+* Dynamo: (???) :warning: Work in progress
+   * `table`
+   * `result`
+
```

Comment:
Sometime it's relevant to add the `method name` for exemple for a metric on Postgre I want to know which method is called